admin/eliotlogging.py

```diff
@@ -0,0 +1,51 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Filter Eliot logs out of Twisted test.log.
+"""
+
+import sys
+
+from flocker.testtools._base import _get_eliot_data
```

Comment:
We should probably not be reaching into a private module to grab a private function. 

Care to make these public before you use them?

(Also probably rename `_get_eliot_data` to something more precise, and document its parameter)